src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,227 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+
+    private static final String VALID_LOGIN = "validLogin";
+    private static final String VALID_PASSWORD = "validPassword";
+    private static final String EXISTING_LOGIN = "existingLogin";
+    private static final String SHORT_LOGIN = "short";
+    private static final String SHORT_PASSWORD = "short";
+    private static final String NULL_LOGIN_MESSAGE = "Login cannot be null.";
+    private static final String NULL_PASSWORD_MESSAGE = "Password cannot be null.";
+    private static final String NULL_AGE_MESSAGE = "Age cannot be null.";
+    private static final String SHORT_LOGIN_MESSAGE = "Login must be at least 6 characters.";
+    private static final String SHORT_PASSWORD_MESSAGE = "Password must be at least 6 characters.";
+    private static final String USER_EXISTS_MESSAGE = "User with this login already exists.";
+    private static final String UNDERAGE_MESSAGE = "User must be at least 18 years old.";
+    private static final int VALID_AGE = 20;
+    private static final int UNDERAGE = 17;
+    private static final int NEGATIVE_AGE = -1;
+
+    private RegistrationService registrationService;
+
+    @BeforeEach
+    public void setUp() {
+        registrationService = new RegistrationServiceImpl();
```

Comment:
This should be done only once, therefore it should be extracted to `@BeforeAll` method.